app/public/stylesheets/sass/_components/_view-container.scss

```diff
@@ -34,4 +34,9 @@
     &:last-child {
       margin-right: 0;
     }
-}
\ No newline at end of file
+}
+
+.streamView_inner {
+  margin-top: 50px;
+  margin-bottom: -50px;
+}
```

Comment:
I'm generally not a big fan of negative margin/padding hacks.
Any particular reason we need this?